plugins/woocommerce/tests/php/includes/class-wc-validation-test.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Validation functions tests
+ *
+ * @package WooCommerce\Tests\Validation.
+ */
+
+/**
+ * Class WC_Validation_Test.
+ */
+class WC_Validation_Test extends \WC_Unit_Test_Case {
+	/**
+	 * Data provider for test_is_postcode().
+	 */
+	public function data_provider_test_is_postcode(): array {
+		$se = array(
+			array( true, '123 45', 'SE' ),
+			array( true, '12345', 'SE' ),
+			array( false, '12 345', 'SE' ),
+			array( false, 'ABC 45', 'SE' ),
+		);
+
+		$li = array(
+			array( true, '9482', 'LI' ),
+			array( true, '9495', 'LI' ),
+			array( false, '8512', 'LI' ),
+			array( false, '0123', 'LI' ),
+			array( false, '948A', 'LI' ),
+		);
```

Comment:
Should the `LI` here be `FI`, instead?

Reply:
Perhaps I misread - looks like this _is_ supposed to be for `LI` country, as the postcode format is different from `FI`

Reply:
Thank you for checking. Indeed, LI did not have tests at all and FI / IT is tested with legacy tests here:
https://github.com/woocommerce/woocommerce/blob/073c3ff3b420d6cdb608a5b27cdff522a08dc8e6/plugins/woocommerce/tests/legacy/unit-tests/util/validation.php#L58-L64